tests/layer_tests/tensorflow_tests/test_tf_FloorDiv.py

```diff
@@ -49,3 +48,65 @@ def test_add_placeholder_const_1D(self, params, ie_device, precision, ir_version
                                                           use_new_frontend=use_new_frontend),
                    ie_device, precision, ir_version, temp_dir=temp_dir,
                    use_new_frontend=use_new_frontend)
+
+
+class TestFloorDivStaticInput(CommonTFLayerTest):
+    def create_flordiv_tf_net(self, min, max, step, y, dtype, ir_version, use_new_frontend, **kwargs):
+        import tensorflow as tf
+        
+        x = np.arange(min, max, step, dtype=dtype)
+        x_shape = x.reshape(kwargs['x_shape']).shape if 'x_shape' in kwargs else x.shape
+
+        tf.compat.v1.reset_default_graph()
+
+        with tf.compat.v1.Session() as sess:
+            x = tf.compat.v1.placeholder(dtype, x_shape, 'Input')
+            y = tf.constant(np.array(y).astype(dtype))
+            res = tf.raw_ops.FloorDiv(x=x, y=y)
+
+            tf.compat.v1.global_variables_initializer()
+            tf_net = sess.graph_def
+
+        ref_net = None
+
+        return tf_net, ref_net
+    
+    def _prepare_input(self, inputs_dict, kwargs):
+        for input in inputs_dict.keys():
+            inputs_dict[input] = np.arange(kwargs['min'], kwargs['max'], kwargs['step'], dtype=kwargs['dtype'])
+            if 'x_shape' in kwargs:
+                inputs_dict[input] = inputs_dict[input].reshape(kwargs['x_shape'])
+        return inputs_dict
+
+    test_inputs = [
+        # test for integers
+        dict(min=-20, max=20, step=1, y=[10],dtype=np.int32),
+        dict(min=-10000, max=10000, step=100, y=[10000], dtype=np.int32),
+        dict(min=-10000, max=10000, step=100, y=[-10000], dtype=np.int32),
+        dict(min=-1e5, max=1e5, step=100, y=[1e5], dtype=np.int32),
+        dict(min=-1e8, max=1e8, step=1e4, y=[1e5], dtype=np.int32),
+        dict(min=-1e8, max=1e8, step=1e4, y=[-1e5], dtype=np.int32),
+        
+        # test for multidimensinal input
+        dict(min=-10000, max=10000, step=10, y=[10000], x_shape=[20, -1], dtype=np.int32),
+        dict(min=-10000, max=10000, step=10, y=[10000], x_shape=[2, 5, -1], dtype=np.int32),
+        dict(min=-10000, max=10000, step=1, y=[10000], x_shape=[2, 5, 10, -1], dtype=np.int32),
+
+        # test for floats
+        dict(min=-20, max=20, step=1, y=[10],dtype=np.float32),
+        dict(min=-10000, max=10000, step=100, y=[10000], dtype=np.float32),
+        dict(min=-10, max=10, step=10, y=[100], dtype=np.float64),
+        dict(min=-1e5, max=1e5, step=100, y=[1e5], dtype=np.float32),
+        dict(min=-1e8, max=1e8, step=1e4, y=[1e5], dtype=np.float32),
+        dict(min=-1e8, max=1e8, step=1e4, y=[-1e5], dtype=np.float32),
```

Comment:
Is it reasonable to use such extreme values here: 10000 and -1e5? Are we sure to avoid sporadic issues?

Reply:
indeed i had quite a big values around `1e6` on FloorDiv input
![image](https://github.com/openvinotoolkit/openvino/assets/5703039/9119da5f-5938-4f60-8d0e-96a185b2866e)

but cases for `1e8` are probably redundant i will replace them with values  `~100-1000`